### PR TITLE
Fixed equations in quadratic-barrier python doc

### PR DESCRIPTION
**Pull request by julesser**

### CODE REVIEW

File: bindings/python/crocoddyl/core/activations/quadratic-barrier.cpp [hunk 1]
```diff
@@ -35,8 +35,8 @@ void exposeActivationQuadraticBarrier() {
       "The activation is zero when r is between the lower (lb) and upper (ub) bounds, beta\n"
       "determines how much of the total range is not activated. This is the activation\n"
       "equations:\n"
-      "a(r) = 0.5 * ||r||^2 for lb < r < ub\n"
-      "a(r) = 0. for lb >= r >= ub.",
+      "a(r) = 0.5 * ||r||^2 for lb >= r >= ub\n"
```
Comment (jcarpent) [inline bindings/python/crocoddyl/core/activations/quadratic-barrier.cpp:38]: lb means lower bound. So the comparison operator is incorrect.

Comment (cmastalli) [reply bindings/python/crocoddyl/core/activations/quadratic-barrier.cpp:38]: Yes, it should be
```cpp
"a(r) = 0.5 * ||r||^2 for lb > r > ub\n"
"a(r) = 0. for lb <= r <= ub.",
```

Comment (julesser) [reply bindings/python/crocoddyl/core/activations/quadratic-barrier.cpp]: Done

**Comment by cmastalli** [comment]
@julesser thank you fixing this error in the documentation :)

**Pull request merged**